go/vt/discovery/healthcheck.go

```diff
@@ -413,6 +416,15 @@ func (hc *HealthCheckImpl) updateHealth(th *TabletHealth, prevTarget *query.Targ
 	defer hc.mu.Unlock()
 
 	tabletAlias := tabletAliasString(topoproto.TabletAliasString(th.Tablet.Alias))
+	// let's be sure that this tablet hasn't been deleted from the authortative map
+	// so that we're not racing to update it and in effect re-adding a copy of the
+	// tablet record that was deleted
+	_, ok := hc.healthByAlias[tabletAlias]
+	if !ok {
+		log.Infof("Tablet %s has been deleted, skipping health update", tabletAlias)
```

Comment:
We should always print the full tablet info instead of just the alias. Makes debugging easier.